sceptre/hooks/__init__.py

```diff
@@ -98,3 +98,33 @@ def decorated(self, *args, **kwargs):
         return response
 
     return decorated
+
+
+def add_stack_hooks_with_aliases(function_aliases: List[str]):
```

Comment:
Isn't it possible to make the original `add_stack_hooks` optionally take a list of aliases in a backwards-compatible way without introducing a new function?

Reply:
Is it possible? Yes. But it would be a bit weird and ugly. A decorator that takes an argument needs to go one level deeper in terms of function scope than a decorator that doesn't. So I'd need to evaluate if the argument is a function or a list and then do different things in each situation. It's just cleaner to define two different functions with explicit names. I don't like fusing together functionality that is better explained if it's separated.